### DISCUSSION TITLE
Basic Coding

### DISCUSSION THREAD

**Discussion by alodagr8** [learning-zone/javascript-interview-questions#31] [General]
Im needing help with the first basic beginners step...

**Comment by enzotlucas** [comment]
Hey there!
I recommend you to look for developing roadmaps that fits for you, like [this](https://github.com/kamranahmedse/developer-roadmap)!
For example, you are looking for js tips, then look at a [frontend roadmap](https://roadmap.sh/frontend), then you only need to search each topic!
I hope it helps you!

**Reply by alodagr8** [reply]
Improvements have been made ..thank you! Any advice yet alone any other suggestions of site references & referrals would be more than appreciated & useful right now.
